lib/default.html

```diff
@@ -0,0 +1,10 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Your Awesome Webpage</title>
+  </head>
+  <body>
+    <p>Make something amazing with the web</p>
```

Comment:
Can you make it clear that his is a bracket's document? Something like "Make something amazing with Brackets" could work